frontend/appflowy_flutter/lib/workspace/presentation/widgets/dialogs.dart

```diff
@@ -206,6 +204,7 @@ class OkCancelButton extends StatelessWidget {
   final String? okTitle;
   final String? cancelTitle;
   final double? minHeight;
+  final MainAxisAlignment? alignment;
```

Comment:
Remove the `?` since this property is assigned a value in the constructor and should not be optional.